lib/ae_mdw/contracts.ex

```diff
@@ -17,6 +17,8 @@ defmodule AeMdw.Contracts do
   alias AeMdw.Util
   alias AeMdw.Validate
 
+  import AeMdwWeb.Helpers.ViewHelper
```

Comment:
Single place to find encoding functions.

Reply:
Do we want for the `AeMdw` app to depend on `AeMdwWeb` app? Shouldn't it be the other way round? Perhaps we can move these functions to a module inside the `AeMdw` app. `Format` maybe?

Reply:
It's a fair point even though all context `AeMdw` modules are already dealing with `AeMdwWeb` concerns like pagination and rendering the view. Moved to `AeMdw.Util.Encoding` until the view is rendered by specific modules for its owner controller replacing the `Format` module that renders for all controllers.